src/App/Components/preview.trip.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import avatar from '../../Assets/images/avatar.png';
+import dateHelper from '../../helper/date.helper';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faMapMarker, faClock, faQuestion } from '@fortawesome/free-solid-svg-icons';
+
+export default ({ request, user }, closeModel) => (
```

Comment:
let us make use of icons as the PO suggested not full color from your card since we already have the design for this in @lemoissonM  PR for getting all requests 
<img width="659" alt="Screen Shot 2019-12-19 at 11 42 11" src="https://user-images.githubusercontent.com/41104288/71162865-a3449700-2254-11ea-905b-b5fb26088956.png">